packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -2,6 +2,64 @@
+
+#### Administrator Roles API third-party admin status update
+
+The [Administrator Roles API](/docs/reference/api/roles/) has been updated to support third-party admin status for a user or a group without requiring a role assignment. See [Role assignment operations](/docs/reference/api/roles/#role-assignment-operations). <!--OKTA-381780-->
+
+#### Custom Group Profile Properties now in Early Access
+
+The [Groups API](/docs/reference/api/groups/) can now manage [custom group profile properties](/docs/reference/api/groups/#custom-profile-properties) after these properties are added to the Group Profile schema. The [Schemas API](/docs/reference/api/schemas) includes a new [Groups Schema object](/docs/reference/api/schemas/#group-schema-object) and [operations](/docs/reference/api/schemas/#group-schema-operations) that support custom group properties. You can use the Schemas API or the Profile Editor in the Admin Console to manage schema extensions. Custom Group Profile properties provide flexibility to manage the default profile for Okta groups in the Okta Admin Console Profile Editor or through the Schemas API. This new functionality simplifies group management and lets you quickly add, edit, or remove custom profile attributes to groups. <!--OKTA-389897-->
+
+#### Domains API is now GA in Preview
+
+The [Domains API](/docs/reference/api/domains/) is now GA in Preview. This API allows you to customize your Okta org domain name for a seamless branded experience so that your users see the same base URL for your application. <!--OKTA-386752-->
+
+#### Domains API supports OAuth 2.0
+
+The [Domains API](/docs/reference/api/domains/) has been updated to support OAuth 2.0. You can grant access to the Domains API using the `okta.domains.manage` and the `okta.domains.read` scopes instead of using SSWS tokens. <!--OKTA-381286-->
+
+#### Okta Sign-In Widget, version 5.6.0
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.6.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-391046-->
+
+#### Refresh Token Rotation is now GA in Preview
+
+[Refresh Token Rotation](/docs/guides/refresh-tokens/refresh-token-rotation/) is now GA in Preview. Refresh Token Rotation helps a public client to securely rotate refresh tokens after each use. When refresh token rotation behavior is enabled in Okta, a new refresh token is returned each time the client makes a request to exchange a refresh token for a new access token.
+<!--OKTA-390933-->
+
+#### SAML parameter SessionNotOnOrAfter now available
```

Comment:
```suggestion
#### SAML parameter SessionNotOnOrAfter now available
```
Either "Generally Available" or "GA", depending on what we decide.

Reply:
Update to "SAML parameter SessionNotOnOrAfter is GA in Preview"